spec/features/authentication/set_up_two_factor_authentication_spec.rb

```diff
@@ -0,0 +1,91 @@
+describe "Set up two factor authentication", type: :feature do
+  let(:organisation) { create(:organisation) }
+  let(:super_admin_organisation) { create(:organisation, super_admin: true) }
+  let(:user) { create(:user, organisations: [super_admin_organisation]) }
+
+  before do
+    sign_in_user(user, pass_through_two_factor: false)
+    visit root_path
+  end
+
+  context "with a super admin user" do
+    it "2FA setup is enforced" do
```

Comment:
Consider rephrasing:
```suggestion
    it "enforces 2FA setup" do
```